lib/themes/dosomething/paraneue_dosomething/js/campaign/prototype.js

```diff
@@ -58,23 +57,16 @@ $(document).on('click', '.js-kudos-button', function(event) {
   const dataKid = $el.attr('data-kid');
   const method = dataKid ? 'DELETE' : 'POST';
 
-  // if (setting('dsKudosReactions.sendToRogue')) {
-  //   makeKudosRequestToRogue(setting('dsKudosReactions.northstarId'), setting('dsKudosReactions.postId'))
-  // } else {
   makeKudosRequest($el, dataKid)
     .done((response) => {
       const kid = get(response, '0.kid', '');
-
-      if (method === 'POST' && !kid) {
-        return;
-      }
-
+      // if (method === 'POST' && !kid) {
+      //   return;
+      // }
```

Comment:
Do we need this block here? I had to comment this out to get the logic correct in this case: 
- Current user has already reacted to a post (reaction button is active). 
- When the current user clicks on the reaction again, the counter increments and the button remains active (looks like it adds another reaction when the user has already reacted instead of undoing the reaction).
- Everything is accurately reflected in the database, it's just the front end that isn't displaying correctly. 
- When you refresh the page, front end is updated and is accurate. 

Why this is happening? 
- I believe it's because the method is `POST` since on a new page, it doesn't know about a `kid` from Rogue. Really, the method should be `DELETE` however, the logic still works since the `/reactions` endpoint in Rogue both creates and soft deletes. 
- Since it deletes, it doesn't send back a `kid` so when this block is not commented out, it just returns before updating the Kudos Interface.